Index: Makefile
===================================================================
--- Makefile
+++ Makefile	2018-12-22 22:47:02.044520609 +0100
@@ -20,8 +20,8 @@
 install-desktop-files:
 	@echo -e '\033[1;32mCreate nedded dirs...\033[0m'
 	mkdir -p $(DESTDIR)$(PREFIX)/share/applications
-	$(INSTALL) 0755 debian/i-nex.desktop $(DESTDIR)$(PREFIX)/share/applications/
-	$(INSTALL) 0755 debian/i-nex-library.desktop $(DESTDIR)$(PREFIX)/share/applications/
+	$(INSTALL_DM) 0755 debian/i-nex.desktop $(DESTDIR)$(PREFIX)/share/applications/
+	$(INSTALL_DM) 0755 debian/i-nex-library.desktop $(DESTDIR)$(PREFIX)/share/applications/
 	
 install-pixmaps:
 
Index: pixmaps/Makefile
===================================================================
--- pixmaps/Makefile
+++ pixmaps/Makefile	2018-12-22 22:48:21.643094683 +0100
@@ -2,10 +2,10 @@
 include ../i-nex.mk
 install:
 	mkdir -p ${DESTDIR}$(PREFIX)/share/pixmaps
-	${INSTALL} 644 i-nex.png $(DESTDIR)$(PREFIX)/share/pixmaps/
-	${INSTALL} 644 i-nex-16.png $(DESTDIR)$(PREFIX)/share/pixmaps/
-	${INSTALL} 644 i-nex-32.png $(DESTDIR)$(PREFIX)/share/pixmaps/
-	${INSTALL} 644 i-nex-128.png $(DESTDIR)$(PREFIX)/share/pixmaps/
+	$(INSTALL_DM) 644 i-nex.png $(DESTDIR)$(PREFIX)/share/pixmaps/
+	$(INSTALL_DM) 644 i-nex-16.png $(DESTDIR)$(PREFIX)/share/pixmaps/
+	$(INSTALL_DM) 644 i-nex-32.png $(DESTDIR)$(PREFIX)/share/pixmaps/
+	$(INSTALL_DM) 644 i-nex-128.png $(DESTDIR)$(PREFIX)/share/pixmaps/
 	
 uninstall:
 
Index: JSON/Makefile
===================================================================
--- JSON/Makefile
+++ JSON/Makefile	2018-12-22 22:45:38.284874078 +0100
@@ -18,7 +18,7 @@
 
 install-bin-i-nex-edid:
 	test -d $(DESTDIR)$(bindir) || mkdir $(DESTDIR)$(bindir)
-	$(INSTALL) 0755 i-nex-edid $(DESTDIR)$(bindir)
+	$(INSTALL_DM) 0755 i-nex-edid $(DESTDIR)$(bindir)
 clean:
 	@echo -e '$(ACTION_COLOR)Clean JSON$(NO_COLOR)'
 	$(RM_COM) $(RMFILE_OPT) i-nex-edid
